Src/java/cql-to-elm/src/main/java/org/cqframework/cql/cql2elm/CqlTranslator.java

```diff
@@ -470,9 +521,42 @@ public CqlErrorListener(LibraryBuilder builder, boolean detailedErrors) {
             this.detailedErrors = detailedErrors;
         }
 
+        private VersionedIdentifier extractLibraryIdentifier(cqlParser parser) {
+            RuleContext context = parser.getContext();
+            while (!(context instanceof cqlParser.LibraryContext)) {
+                context = context.parent;
+                if (context == null) {
+                    break;
+                }
+            }
```

Comment:
This suggestion is so trivial that I hesitate to make it, so feel free to ignore it if you wish, but... I personally prefer putting nullchecks into the condition versus a dedicated `break`.  E.g.,
```java
            while (context != null && !(context instanceof cqlParser.LibraryContext)) {
                context = context.parent;
            }
```

Reply:
Agreed and applied